package.json

```
@@ -16,8 +16,8 @@
     "./react": "./tsconfig.react.json"
   },
   "engines": {
-    "npm": ">=8",
-    "node": ">=16"
+    "npm": "^8",
+    "node": "^16"
```

We should keep this here

```suggestion
    "npm": ">=8",
    "node": ">=16"

---

Good catch! Thank you